frog/imports/client/Wiki/index.js

```diff
@@ -130,27 +138,30 @@ class WikiComp extends React.Component<WikiCompPropsT, WikiCompStateT> {
     });
 
     window.wikiDoc = this.wikiDoc;
-
-    Mousetrap.bindGlobal('ctrl+n', () =>
-      this.setState({ createModalOpen: true })
-    );
-    Mousetrap.bindGlobal('ctrl+s', () => this.setState({ docMode: 'view' }));
-    Mousetrap.bindGlobal('ctrl+e', () => this.setState({ docMode: 'edit' }));
-    Mousetrap.bindGlobal('ctrl+f', () =>
-      this.setState({ findModalOpen: true })
-    );
+    if (!this.props.embeddedPage) {
+      Mousetrap.bindGlobal('ctrl+n', () =>
+        this.setState({ createModalOpen: true })
+      );
+      Mousetrap.bindGlobal('ctrl+s', () => this.setState({ docMode: 'view' }));
+      Mousetrap.bindGlobal('ctrl+e', () => this.setState({ docMode: 'edit' }));
+      Mousetrap.bindGlobal('ctrl+f', () =>
+        this.setState({ findModalOpen: true })
+      );
+    }
   }
 
   componentDidUpdate(prevProps) {
-    const pageTitle = decodeURIComponent(this.props.match.params.pageTitle);
+    const pageTitle =
+      decodeURIComponent(this.props.match?.params.pageTitle) ||
+      this.props.embeddedPage.pageId;
```

Comment:
We are setting this.props.embeddedPage.pageId to pageTitle

Reply:
It was an error, `embeddedPage` should have the string `pageTitle` not `pageId`